### PR TITLE
Phpdoc 2

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

**Comment by stojg** [comment]
Thanks! would you mind rebase the three commits into one commit?

File: code/tasks/SyncProjectsAndEnvironments.php [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 class SyncProjectsAndEnvironments extends BuildTask {
 
+	/**
+	 * @param  SS_HTTPResponse $request
```
Comment (halkyon) [inline code/tasks/SyncProjectsAndEnvironments.php]: Double space between param and SS_HTTPResponse? :)

Comment (spekulatius) [inline code/tasks/SyncProjectsAndEnvironments.php]: Sorry, I'll fix that - including the multiple commits. Give me a sec.

**Comment by spekulatius** [comment]
Hello @stojg and @halkyon ,

Thanks for your reply. I fixed this - finally. Sorry for the multiple commits and the double spaces. Shouldn't be like this.

Cheers

**Pull request merged**